jpo-asn-runtime/src/test/java/us/dot/its/jpo/asn/runtime/examples/AInteger.java

```diff
@@ -0,0 +1,32 @@
+package us.dot.its.jpo.asn.runtime.examples;

```

Comment:
suggestion: for this examples directory it would be nice to either update the README to include some descriptions about purpose or to add a README to this directory for the same reason. It seems like these are only included as samples of implementing the actual classes?

Reply:
I added a `package-info.java` file to this directory with a javadoc documenting that the purpose of these examples is for the unit tests to use.